.readthedocs.yaml

```diff
@@ -6,6 +6,12 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.12"
+  jobs:
+    post_install:
+      - pip install cmake pybind11[global]
+      - cmake -B build -DCMAKE_INSTALL_PREFIX=$(pwd)
+      - cmake --build build
+      - cmake --install build
```

Comment:
imho it can be worth leaving a comment here on what the purpose of this step is, e.g., "These steps are required for building the documentation on ReadtheDocs".